README.md

```diff
@@ -1,7 +1,31 @@
-# OOP-school-library
+![](https://img.shields.io/badge/Microverse-blueviolet)
+ #  School Library
 Imagine that you are a librarian of a University, this app allows you to record what books are in the library and who borrows them.
+ 
 
-## Authors:
+ ## Screenshot
+
+![screenshot](./a.png)
+
+ 
+ ## Built With
+
+ - Ruby
+
+ ## Getting Started
+
+ To get a local copy up and running follow these simple example steps.
+
+ `https://github.com/Kakalanp/OOP-school-library.git`
+
```

Comment:
- [ ] For the getting started, we just have a URL, there are no instructions for cloning the budget and navigating to the root directory of the project. For example:
```
git clone <URL>
cd <name of repository here>
open with a code editor
```

#### Kindly implement this 👍

Reply:
Noted.
The changes you mentioned have been implemented.